stablehlo/dialect/StablehloOps.td

```diff
@@ -1222,7 +1222,13 @@ def StableHLO_WhileOp: StableHLO_Op<"while", [
     }
   }];
   let hasCustomAssemblyFormat = 1;
-  let hasVerifier = 1;
+  
+  // TODO: Remove this after https://github.com/openxla/stablehlo/pull/145 landed.
```

Comment:
Let me try, removing this directly will lead to error in my local branch (because checkout it before the refactor in main branch), I I assume the PR should be fine.

Reply:
Rebase work perfectly! The downside is that I have to force git push to 
[zhouxin913/stablehlo](https://github.com/zhouxin913/stablehlo) which may break the location of this comment.

Instead of removing, unfortunately, I have to add this method in the `extraClassDeclaration` section lines above, as discussed in https://github.com/openxla/stablehlo/pull/145#discussion_r974713946